Sources/GtkCodeGen/GtkCodeGen.swift

```diff
@@ -224,6 +226,14 @@ struct GtkCodeGen {
                 return false
             }
 
+            //can cause problems with gtk versions older than 4.20.0
+            guard
+                member.cIdentifier != "GTK_PAD_ACTION_DIAL",
+                member.name != "GTK_PAD_ACTION_DIAL"
```

Comment:
For this check I meant that you should check that the enumeration name is "PadActionDial" when checking the member cIdentifier. I realise that the member names are prefixed so there won't be clashes, but it's probably nice for future refactorers to have the enumeration name check there so they know which enum the rule is meant to apply to.

```swift
guard enumeration.name != "PadActionDial" || member.cIdentifier != "GTK_PAD_ACTION_DIAL" else {
    return false
}
```

(the enum name may be `GtkPadActionDial` instead of `PadActionDial`, not sure)